tests/shh-module/test_shh_filter.py

```diff
@@ -0,0 +1,21 @@
+import random,string,gevent
+
+def test_shh_filter(web3,skip_if_testrpc):
```

Comment:
*nitpick*

Can you add a space after the `,` here and in the other test cases as well.